hr_payslip_move_date/models/hr_payslip.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+# © 2014 Eficent
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import fields, models, api
+
+
+class HrPayslip(models.Model):
+    _inherit = 'hr.payslip'
+
+    move_date = fields.Date(
+        string='Force move date',
+        required=False,
+    )
+
+    @api.onchange('move_date')
+    def onchange_move_date(self):
+        period_obj = self.env['account.period']
+        if self.move_date:
+            period_ids = period_obj.find(dt=self.move_date)
+            if period_ids:
+                self.period_id = period_ids[0]
```

Comment:
@feketemihai it already controlled by account.period find method. I tested it on runbot. When i selected 2017-03-30 as force date, warning showed

![selection_004](https://cloud.githubusercontent.com/assets/1947376/14731700/8f264ec0-087d-11e6-9704-8e45d35e243c.png)